app/styles/_color.scss

```diff
@@ -60,6 +60,7 @@ body {
   --divider-dark: #e9e9e9;
 
   --border-radius: 2px;
+  --border-radius-3: 3px;
```

Comment:
do we require this border radius ? and is there any visible difference

Reply:
Removed.